iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/HttpsRequestTests.java

```diff
@@ -1,47 +0,0 @@
-/*
- *  Copyright (c) Microsoft. All rights reserved.
- *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
- */
-
-package tests.integration.com.microsoft.azure.sdk.iot.iothub;
-
-import com.microsoft.azure.sdk.iot.device.exceptions.TransportException;
-import com.microsoft.azure.sdk.iot.device.transport.https.HttpsMethod;
-import com.microsoft.azure.sdk.iot.device.transport.https.HttpsRequest;
-import com.microsoft.azure.sdk.iot.device.transport.https.HttpsResponse;
-import org.junit.Test;
-import tests.integration.com.microsoft.azure.sdk.iot.helpers.IntegrationTest;
-import tests.integration.com.microsoft.azure.sdk.iot.helpers.annotations.IotHubTest;
-
-import java.io.IOException;
-import java.net.URL;
-
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.Matchers.startsWith;
-import static org.junit.Assert.assertThat;
-
-/** Integration tests for HttpsRequest. */
-@IotHubTest
-public class HttpsRequestTests extends IntegrationTest
-{
-    @Test
-    public void sendHttpsRequestGetsCorrectResponse() throws IOException, TransportException
```

Comment:
If the HttpsRequest class wasn't working as expected, other tests would catch it. This test is nowhere near as exhaustive as our basic telemetry over HTTP tests